app/views/cases/overturned_foi/new.html.slim

```diff
@@ -0,0 +1 @@
+= render partial: 'cases/overturned_sar/new', locals: { kase: @case }
```

Comment:
Is this correct? Do we need `overturned_foi/new`?

Reply:
@misaka I think this is correct we are doing overturned SARs not FOI

Reply:
We have two correspondence types: overturned_sar, and overturned_ico.  As the view path is determined by the correspondence type abbreviation we need two files, one int he overturned_foi directory, and one in the overturned_sar directory.  But they just include the correct partials.  the other way of doing it was to have a special method on correspondence type (or case) that returns the view path to use, and change everywhere we generate the virew path using the type abbreviation to use this new method.  I chose the path of least resistance.

Reply:
yep, `reply_method` was introduced for SARs, and its meaning is slightly different.

`delivery_method` refers to the way in which the case was sent to us, and can have values `sent_by_post` and `sent_by_email` (past tense).
`reply_method` refers to the way in which the response will be sent, and can have the values `send_by_post` and `send_by_email`

Reply:
After discussing it seems that this should be in shared.